src/components/reader/PdfLayoutScan.module.css

```
@@ -0,0 +1,211 @@
+  font-family: var(--font-display), system-ui, sans-serif;
+  font-size: 10px;
+  font-weight: 700;
+  line-height: 1.25;
+  letter-spacing: 0.01em;
+  white-space: nowrap;
+  color: var(--background);
+  background: var(--accent);
+  box-shadow: 0 2px 10px color-mix(in srgb, var(--accent) 50%, transparent);
+  animation: tagIn 0.34s var(--ease);
+}
+@keyframes tagIn {
```

Stylelint (17.12.0)

[error] Expected keyframe name "tagIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)